components/cards/WhatWeDoCard.jsx

```diff
@@ -0,0 +1,42 @@
+import { motion, useAnimation } from "framer-motion";
+import { useInView } from "react-intersection-observer";
+import { useEffect } from "react";
+import Image from 'next/image'
+import { H4 } from '../ui/Headings';
+
+const WhatWeDo = ({index, title, description, image}) => {
```

Comment:
One more thing, it's a good practice to have the component name the same as the file name.

Reply:
Applied